### DISCUSSION TITLE
Suggestion: Display the date last modified of a form

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-freeform#482] [Feature Requests]
Filing 3 separate suggestions following feedback from a client. We have a large multi-site Craft install with 8 sites. As the website continues to grow, our client is noticing some growing pains related to Freeform.

Suggestion 3: Forms should behave more like Craft's native functionality and provide information on the date a form was last modified. It feels like Solspace have often tried to re-invent the wheel instead of using Craft's native functionality, but it would be great if we could see some of the basic information on the forms like when it was last updated.

**Comment by kjmartens** [comment]
Thanks for your feedback, @darylknight. We're in the process of making some big improvements to Freeform, so it's very possible features/info like this will be something we can easily implement. 🙂

**Comment by kjmartens** [comment]
This will be added in the next major version of Freeform. 🙂

**Comment by kjmartens** [comment]
This has been implemented in Freeform 5 (now in beta). 🙂
